.github/workflows/maven.yml

```diff
@@ -19,8 +19,8 @@ on:
 
 jobs:
 
-  Asm-tests:
-    name: Asm tests
+  Ck-armv7-tests:
+    name: Ck armv7 tests
```

Comment:
Lets use `Concurrency Kit` for the name description (you can keep `ck` for the previous line). Same applied to CK-armv8